packages/test/test-service-load/src/optionsMatrix.ts

```diff
@@ -41,7 +41,9 @@ export function generateRuntimeOptions(seed: number) {
         disableIsolatedChannels: booleanCases,
         generateSummaries: [true],
         initialSummarizerDelayMs: numberCases,
-        summaryConfigOverrides: [undefined],
+        summaryConfigOverrides: [{
+            maxAckWaitTime: 10 * 60 * 1000,
```

Comment:
I'm not sure it should be here. In a sense that our stress tests should test what is in production, and not overwrite it.

Reply:
it turns out he server config is 10 min which was getting clobbered by maxSummarizeAckWaitTime, so we shouldn't need this.